app/src/main/java/com/android/streetworkapp/model/workout/WorkoutRepositoryFirestore.kt

```diff
@@ -161,6 +164,91 @@ class WorkoutRepositoryFirestore(private val db: FirebaseFirestore) : WorkoutRep
       Log.e(ERROR_TAG, "Error saving WorkoutData for UID=$uid: ${e.message}")
     }
   }
+  /**
+   * Sends a pairing request from one user to another.
+   *
+   * @param fromUid The UID of the user sending the request.
+   * @param toUid The UID of the user receiving the request.
+   */
+  override suspend fun sendPairingRequest(fromUid: String, toUid: String) {
+    val pairingRequest =
+        PairingRequest(
+            requestId = db.collection("pairingRequests").document().id,
```

Comment:
The "pairingRequest" could be a constant, as it is used multiple times in the file

Reply:
Done